perses/analysis/analysis.py

```diff
@@ -229,12 +261,24 @@ def plot_work_trajectories(pdf, work, title=""):
                     pdf.savefig()  # saves the current figure into a pdf page
                     plt.close()
 
-                if len(work) > 0:
-                    # Plot work for all chemical transformations.
-                    plot_work_trajectories(pdf, work, title='(all transformations)')
-
-                    # Plot work separated out for each chemical transformation
-                    #[niterations, nsteps] = work.shape
-                    #transformations = dict()
-                    #for iteration in range(niterations):
-                    #    plot_work_trajectories(pdf, work, title='(all transformations)')
+                for modname in ['NCMCEngine', 'NCMCHybridEngine']:
+                    work = dict()
+                    for direction in ['delete', 'insert']:
+                        varname = '/' + envname + '/' + modname + '/' + 'work_' + direction
```

Comment:
Has this been run with the  benchmark? Because it shouldn't work, hence why I had `modname = envname`